### PR TITLE
more use case linking

### LINKED ISSUE #151: Link to package docs in use cases table
Would it be possible to link to the package docs in the [use cases table](https://ropensci.org/usecases/)? Since the docs are always docs.ropensci.org/resource, this should be pretty straightforward?

### PR DESCRIPTION
**Pull request by maelle**
Fix #151

### CODE REVIEW

File: get_use_cases.R [hunk 1]
```diff
@@ -76,11 +73,39 @@ get_info <- function(id, packages = packages) {
...
   auto_unbox = TRUE
   )
+
+link_resource <- function(resource, packages = get_packages()) {
+  resource <- trimws(resource)
+  resource <- sub(".$", "", resource)
+  if (resource %in% packages) {
+    return(sprintf("[%s](https://docs.ropensci.org/%s)", resource, resource))
+  }
+  if (resource == "rOpenSci package development guide book") {
```
Comment (maelle) [inline get_use_cases.R:91]: this and the following lines mean the use cases need to mention the books with these exact phrases, so edits of the Discourse posts might be necessary. Does this sound ok?

File: get_use_cases.R [hunk 2]
```diff
@@ -76,11 +73,39 @@ get_info <- function(id, packages = packages) {
...
+  }
+  if (resource == "rOpenSci contributing guide book") {
+    return("[rOpenSci contributing guide book](https://contributing.ropensci.org)")
+  }
+  if (resource == "rOpenSci blog guide book") {
+    return("[rOpenSci blog guide book](https://blogguide.ropensci.org)")
+  }
+  if (resource == "HTTP testing in R book") {
+    return("[HTTP testing in R book](https://books.ropensci.org/http-testing)")
+  }
```
Comment (maelle) [inline get_use_cases.R:102]: We can add more as needed.

**Comment by steffilazerte** [comment]
Looks great! Sorry it took me so long to review 🤦🏼

**Comment by maelle** [comment]
Thank you and no problem! The only person allowed to complain is the person who wanted the feature... you :joy:

**Comment by steffilazerte** [comment]
Exactly why I should have been faster! Thank you 😁

**Pull request merged**